test_profilehooks.py

```diff
@@ -180,12 +180,9 @@ def doctest_coverage_when_source_is_not_available(self):
 
 
         >>> run_exitfuncs()
+
+        >>> print(buffer.getvalue().strip())
         <BLANKLINE>
-        *** COVERAGE RESULTS ***
-        sample_fn (<doctest test_profilehooks.doctest_coverage_when_source_is_not_available[0]>:1)
-        function called 2 times
-        <BLANKLINE>
-        cannot show coverage data since co_filename is None
```

Comment:
This is wrong, please revert.  `@coverage` should be printing coverage information, like the test used to assert.